src/symfony/src/Resources/config/doctrine-mapping/PublicKeyCredentialSource.orm.xml

```diff
@@ -5,7 +5,7 @@
     xsi:schemaLocation="http://doctrine-project.org/schemas/orm/doctrine-mapping https://raw.github.com/doctrine/doctrine2/master/doctrine-mapping.xsd"
 >
     <mapped-superclass name="Webauthn\PublicKeyCredentialSource">
-        <field name="publicKeyCredentialId" type="base64" unique="true"/>
+        <field name="publicKeyCredentialId" type="base64" unique="true" length="250"/>
```

Comment:
FYI: The maximum length is specified as 1023.

see: https://github.com/w3c/webauthn/pull/1664

Reply:
Indeed. Many thanks for the hint.
As it is base64 encoded and to prevent any issue, 2048 seems to be valid